extensions/http/jersey/src/main/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/EdcApiExceptionMapper.java

```diff
@@ -63,13 +61,17 @@ public Response toResponse(Throwable exception) {
 
         var status = exceptionMap.getOrDefault(exception.getClass(), SERVICE_UNAVAILABLE);
 
-        if (exception instanceof EdcApiException && verboseResponse) {
+        if (exception instanceof EdcApiException) {
             var edcApiException = (EdcApiException) exception;
-            var apiError = ApiErrorDetail.Builder.newInstance()
+
+            var errorDetail = ApiErrorDetail.Builder.newInstance()
                     .message(edcApiException.getMessage())
                     .type(edcApiException.getType())
                     .build();
-            return Response.status(status).entity(List.of(apiError)).build();
+
+            return Response.status(status)
+                    .entity(List.of(errorDetail))
+                    .build();
```

Comment:
Not necessarily related to this PR. But we do not provide the error details (in the `entity` part of the Response) is the exception is not of type `EdcApiException`

Reply:
@bscholtes1A I think this is correct, as the other exceptions should cause a 5xx error (a server fault, so the client won't need any detail). 
I added an issue on this topic: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1730